templates/frontend/component/quantities/grid-layout.php

```diff
@@ -101,15 +101,15 @@ class="ppom-quantity"
 						data-includeprice="<?php echo esc_attr( $include_productprice ); ?>"
 						data-usebase_price="<?php echo esc_attr( $usebaseprice ); ?>"
 						data-unlink_qty="<?php echo esc_attr( $unlink_qty ); ?>"
-						<?php echo apply_filters( 'ppom_fe_form_element_custom_attr', '', $fm ); ?>
+						<?php echo apply_filters( 'ppom_fe_form_element_custom_attr', '', $fm ); // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped ?>
 						data-price="<?php echo esc_attr( $the_price ); ?>"
```

Comment:
we should not have ignore announcement for the apply_filters call. We cannot rely on the data which comes by the hook.

So, we should surround it with a proper escaping function.

That's valid for all `ppom_fe_form_element_custom_attr` apply_filters call in the codebase. (There are near to 23 results)